RoboJackets/robocup-software
# Role Assignment Unassigned Functionality

## Description
Created unassigned_cost_fn functions for Cost_fn and functionality for them in role assignment. Also added a case to ignore none visible robots in terms of role assignment.

## Steps to test
### Test Case 1
1. Set the unassigned_pass_cost of pass cost in pass tactic to -9999
2. Open sim
3. Run pass play

Expected result: Nothing should happen, switching the cost to be positive and large again should result in normal play behavior.
